kafka-rest/pom.xml

```diff
@@ -153,6 +153,11 @@
            <artifactId>bcprov-ext-jdk15on</artifactId>
            <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
nit: Should this actually be `hamcrest-all` or it can be just `hamcrest-library`? Unfortunately we seem to be using 1.x - Hamcrest 2.x packages everything in a single JAR.